library/core/src/panicking.rs

```diff
@@ -49,6 +50,7 @@ pub const fn panic(expr: &'static str) -> ! {
 
 #[inline]
 #[track_caller]
+// #[rustc_const_unstable(feature = "core_panic", issue = "none")]
```

Comment:
seems like a leftover

Reply:
That was definitely me trying to figure out the previous blocker. I'll remove this line when I'm on my laptop.

Reply:
I think I actually screwed up the rebasing before. Diff should be good now.